editor/components/block-list/style.scss

```diff
@@ -934,3 +934,30 @@
 		content: '←';
 	}
 }
+
+.editor-block-list__block {
+	@include break-small {
+		// Increase the hover and selection area around blocks.
+		// This makes the blue hover line and the settings button appear even if
+		// the mouse cursor is technically outside the block.
+		// This improves usability by making it possible to click somewhat outside
+		// the block and select it. (eg. A fuzzy click target.)
+		&:before {
+			bottom: 0;
+			content: '';
+			left: -$parent-block-padding - $block-padding;
```

Comment:
I really appreciate you making this change, and the intent behind it. 

It introduced a small regression, though, as it makes a horizontal scrollbar appear when a block is full-wide. 

I discovered this while rebasing #7365. I will implement a fix in that branch, perhaps you can help me review it so we can get it in faster? Thanks.